### PR TITLE
Make bean item details more responsive and display all the info

### PR DESCRIPTION
**Pull request by ghmeier**
Looks like this:

update

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
         <title>Expresso</title>
     </head>
     <body>
-        <div id="root" class="h-100"></div>
+        <div id="root" class="h-100 helvetica"></div>
```
Comment (jonnykry) [inline public/index.html:36]: ooo la-la

File: src/components/dashboard/browse/BeanItemDetails.js [hunk 2]
```diff
@@ -1,45 +1,65 @@
...
 
 class BeanItemDetails extends Component {
     render() {
+        const subClass = 'pointer f2 dim br1 ba bw1 tc ph2 mh4 pv3';
+        const subscribe = this.props.bean.inStockBags > 0 && this.props.bean.isActive ?
+            <Link to={'/dashboard/subscribe/' + this.props.params.id} className="no-underline black">
+                <div className={subClass + ' green'}>Subscribe</div>
+            </Link> :
+                <div>
+                    <div className={subClass + ' black-40 black--40'}>Not Available</div>
```
Comment (jonnykry) [inline src/components/dashboard/browse/BeanItemDetails.js:16]: This is cool - will it hook into when people subscribe or do Roasters need to manually update it?

**Review by jonnykry — APPROVED** [review]
LGTM I like the look of the preview!

File: src/components/dashboard/browse/BeanItemDetails.js (hunk 2 above)
Comment (ghmeier) [reply src/components/dashboard/browse/BeanItemDetails.js:16]: roasters manually update `isActive`, but the `bagsInStock` should decrement correctly when orders are created.

**Pull request merged**